src/api.rs

```diff
@@ -244,7 +252,8 @@ pub fn api(root: &Path) -> anyhow::Result<Router> {
         .route("/packit/metadata", get(get_metadata_since))
         .route("/file/:hash", get(get_file).post(add_file))
         .route("/packet/:hash", post(add_packet))
-        .route("/git/fetch", post(git_fetch))
+        .route("/git/fetch", get(git_fetch))
```

Comment:
why are we changing from `POST` to `GET` here? I thought we used POST on purpose for some reason (perhaps to indicate it changes the server contents)

Reply:
there was no body just felt like get would do the job but think about it this is more in the flavour of a PUT request, but for now will revert it back